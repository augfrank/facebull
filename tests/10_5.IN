M0   C0   C1  2059
M1   C0   C2  2455
M2   C0   C4  2078
M3   C0   C5  2886
M4   C0   C6  2855
M5   C0   C7  2612
M6   C1   C0  2397
M7   C1   C2  2667
M8   C1   C3  2680
M9   C1   C5  2436
M10   C1   C6  2761
M11   C1   C7  2084
M12   C1   C8  2076
M13   C2   C0  2693
M14   C2   C1  2810
M15   C2   C4  2776
M16   C2   C5  2952
M17   C2   C8  2154
M18   C3   C2  2408
M19   C3   C5  2208
M20   C3   C6  2601
M21   C3   C7  2574
M22   C3   C9  2242
M23   C4   C0  2862
M24   C4   C3  2695
M25   C4   C5  2662
M26   C4   C6  2283
M27   C4   C8  2241
M28   C5   C0  2260
M29   C5   C2  2639
M30   C5   C3  2167
M31   C5   C6  2785
M32   C5   C8  2457
M33   C6   C0  2712
M34   C6   C1  2216
M35   C6   C2  2850
M36   C6   C4  2961
M37   C6   C7  2002
M38   C7   C0  2718
M39   C7   C4  2636
M40   C7   C5  2261
M41   C7   C6  2129
M42   C7   C9  2177
M43   C8   C1  2781
M44   C8   C2  2322
M45   C8   C3  2431
M46   C8   C6  2025
M47   C8   C7  2718
M48   C8   C9  2973
M49   C9   C1  2727
M50   C9   C3  2327
M51   C9   C5  2082
M52   C9   C6  2530
M53   C9   C7  2006
