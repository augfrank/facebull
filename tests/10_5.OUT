22171
2 11 13 18 27 30 34 42 46 51
